README.md

```diff
@@ -1 +1,161 @@
-README
\ No newline at end of file
+# Job Manager
+
+## Summary
+
+Worker service which runs Linux processes.
+[![CI](https://github.com/alxmke/jobs/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/alxmke/jobs/actions/workflows/ci.yml)
+
+## Outline
+
+### Library
+
+The Worker library is a Go package which makes Linux OS calls that manage a given set of processes. A Worker makes calls which start and stop jobs, stream and process their output, and handle errors.
+
+Workers will keep their process statuses in a map in memory, which updates as the status of any given process changes. Jobs are tracked and queried using this in-memory map, and if this map and its Worker service  terminates, then so too consequently will the tracking of and control over its jobs.
+
+As a convenience, the Worker writes the process output to a log file on disk in the /tmp directory (by default, configurable). Should conditions requrie it, due to spacial and other scale considerations, log files can be handled with more complex policies. As desired/required, any of the following could be adopted: logs can be compressed and/or archived locally, or stored using a cloud solution like AWS S3, or safely pruned based on some given criteria (date, or available space, for instance), or any combination of these (provided as optionals).
+
+Resource control will be managed using the [cgroups](https://pkg.go.dev/github.com/containerd/cgroups) Go package.
+
+```golang
+// Command is a job request with given command name and arguments.
+type Command struct {
+	Name      string               // name/path of command
+	Args      []string             // command arguments
+	Resources specs.LinuxResources // resource control for process
+}
+
+// Status of the process.
+type Status struct {
+	Pid      int  // process ID
+	ExitCode int  // exit code of process, -1 if not exited or if terminated by signal
+	Exited   bool // has the process exited
```

Comment:
I think you've covered both, exited on it's own and exited due to being stopped, but to clarify, can you explain how both states will be represented?